packages/codemirror-lang-lxlquery/src/syntax.grammar

```diff
@@ -47,15 +47,15 @@ freetext {
 
   CompareOperator { ">" | "<" | ">=" | "<=" }
 
-  booleanOperator { "AND" | "OR" | "NOT" }
+  BooleanOperator { "AND" | "OR" | "NOT" }
```

Comment:
Exporting `BooleanOperator` (oddly enough) breaks the highlighting of BooleanQuerys as it is implemented now. But of course we should do it.

My suggestion is when this and https://github.com/libris/lxlviewer/pull/1186 is merged, we properly re-add (the option to) highlighting operators within a valid BooleanQuery.